cmd/templ/lspcmd/proxy/server.go

```diff
@@ -1216,3 +1256,24 @@ func (p *Server) Request(ctx context.Context, method string, params interface{})
 	defer p.Log.Info("client -> server: Request end")
 	return p.Target.Request(ctx, method, params)
 }
+
+func mapToSymbol(m map[string]interface{}) (interface{}, error) {
```

Comment:
Unfortunate that we have to do this! But I can't think of a way around it either.

Reply:
Indeed, this is just how they define the protocol, the best way is to put it in a custom marshaler, however, here we have already received the data in the `map[string]any`